From: Harald Hoyer <harald@redhat.com>
Date: Fri, 8 Aug 2014 16:54:13 +0200
Subject: Added arch tuple for PPC64LE

Thanks to Brent Baude <bbaude@redhat.com>, who checked with the debian
guys, that this is correct and provided the patch.
---
 src/shared/architecture.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/shared/architecture.h b/src/shared/architecture.h
index 4821d5d..ee50ff4 100644
--- a/src/shared/architecture.h
+++ b/src/shared/architecture.h
@@ -85,7 +85,7 @@ Architecture uname_architecture(void);
 #    define LIB_ARCH_TUPLE "ppc64-linux-gnu"
 #  else
 #    define native_architecture() ARCHITECTURE_PPC64_LE
-#    error "Missing LIB_ARCH_TUPLE for PPC64LE"
+#    define LIB_ARCH_TUPLE  "powerpc64le-linux-gnu"
 #  endif
 #elif defined(__powerpc__)
 #  if defined(WORDS_BIGENDIAN)
